playbooks/common/openshift-cluster/upgrades/v3_3/master_config_upgrade.yml

```diff
@@ -48,3 +48,13 @@
     dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
     yaml_key: 'controllerConfig.servicesServingCert.signer.keyFile'
     yaml_value: service-signer.key
+
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'admissionConfig.pluginConfig'
+    yaml_value: "{{ openshift.master.admission_plugin_config }}"
+
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'kubernetesMasterConfig.admissionConfig'
+    yaml_value:
```

Comment:
Is this also supposed to happen on 3.3 upgrades?

If this is something we want to do every upgrade going forward if release is gte 3.3, we probably don't want hooks and instead should plug it into https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml#L57 and have it run if the gta 3_3 var is set.

If it's one time 3.4 only, then the hooks look correct but suspect this 3.3 block could go away.

If it's literally just run for 3.3 and 3.4 upgrades only, then this is all correct.